tests/integration/operator/operator_test.go

```diff
@@ -0,0 +1,88 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integrationoperator
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Operator", func() {
+	BeforeEach(func() {
+		// Add there code to run before each test if needed
+	})
+
+	When("a fresh cluster exist", func() {
+		It("the operator can be installed", func() {
+			By("using the helm chart with default values")
+			GinkgoWriter.Println("Deploying Operator using default helm charts located in /chart folder")
+
+			// This is only for downstream testing, because the operator will be installed previously in CI pipeline
+			if skipDeploy == "true" {
+				Skip("Skipping the deployment of the operator and the tests")
+			} else {
+				deployOperator()
+			}
+
+			Expect(operatorIsRunning()).To(Equal(true))
+		})
+	})
+
+	When("the operator is installed", func() {
+		Context("a control plane can be installed and uninstalled", func() {
+			istioVersions, err := getIstioVersions("/work/versions.yaml")
+			if err != nil {
+				Fail(fmt.Sprintf("Error getting istio versions from version.yaml file: %v", err))
+			}
+
+			It("for every istio version in version.yaml file", func() {
+				for _, version := range istioVersions {
+					fmt.Print("\nDeploying Istio Control Plane for version: ", version)
```

Comment:
I'm using fmt.Print in two places only because if we do not use it we do not see anything in the result of the test execution, GinkgoWriter is only displayed in the console when the test fails. We can print everything with GinkgoWriter if we run the test instead of using `go test` by using ginkgo command

Reply:
and if we add `-v` to `go test`? maybe that'll enable the printing

Reply:
or we just use `ginkgo` to run the tests. we could do that by using `go run github.com/onsi/ginkgo/v2/ginkgo tests/integration/operator/` -- that way we don't need to install the binary

Reply:
Done, I'll push the changes later with the changes from Marko comments

Reply:
Done this changes